src/main/webapp/css/srp_style.css

```diff
@@ -22,6 +22,19 @@
   transition: box-shadow .25s;
   border-radius: 2px;
   width: 400px;
+  height: 260px;
```

Comment:
Change this to max-height, so cards that don't fill up 260px aren't necessarily extended.